src/resources/filters/quarto-post/book.lua

```diff
@@ -1,6 +1,9 @@
 -- book.lua
 -- Copyright (C) 2020-2022 Posit Software, PBC
 
+local authors = require '../common/authors'
+local license = require '../common/license'
+
```

Comment:
🤩

Reply:
This has caused me to enter LUA inlining nightmare land. See next commit, then following failure:

```
too many local variables (limit is 200) in main function near '='
[326](https://github.com/quarto-dev/quarto-cli/actions/runs/4833870917/jobs/8614410733?pr=5348#step:5:327)
```

![salute](https://user-images.githubusercontent.com/261654/235231533-2661174e-8e88-4795-9c2d-b25c7c420200.gif)

Reply:
That's not terrifying at all